### PR TITLE
Change chain name to XPLA Chain

### PR DESCRIPTION
**Pull request by caramis**

### CODE REVIEW

**Review by psy2848048 — APPROVED** [review]

File: content/docs/develop/tools/xpla-js/_index.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ title: Xpla.js
```
Comment (nowooj) [inline content/docs/develop/tools/xpla-js/_index.md:7]: title: Xpla.js

File: content/_index.md [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
+XPLA Chain is an [open-source blockchain](https://github.com/xpladev/xpla) hosting a vibrant ecosystem of decentralized applications (dApps) and top-tier developer tools. XPLA Chain is based on [Tendermint](https://tendermint.com/), a popular blockchain engine based on Byzantine Fault Tolerant (BFT) consensus which is robust against double-spend attacks and is tolerant against a set of up to 1/3. Through [IBC](https://ibc.cosmos.network/) and [COSMOS SDK](https://docs.cosmos.network/), the XPLA Chain can be easily connected with other various chains and is a developer-friendly chain. Using proof-of-stake conesensus, [Ethereum Virtual Machine (EVM)](https://ethereum.org/en/developers/docs/evm/) Compatibility, XPLA Chain SDK, a software kit for the game developers, and XPLA Vault would give the users extensive experiences of De-Fi, Play to Own (P2O) gaming, and lastly, would lead to the adoption of Web 2 users to Web 3 space. Also, XPLA, all capital letters of the blockchain name, is the XPLA Chain’s native staking token. XPLA is used for [governance]({{< ref "glossary#governance" >}}) and [staking]({{< ref "glossary#staking" >}}).
```
Comment (jhlee-young) [inline content/_index.md]: conesensus -> consensus

Comment (jhlee-young) [reply content/_index.md]: and I guess `, all capital... blockchain name,` can be removed with this update.

Comment (caramis) [reply content/_index.md]: fixed !

File: content/docs/develop/tools/wallet-provider/get-started-with-wallet-provider.md [hunk 3]
```diff
@@ -39,7 +39,7 @@ If you're using a frontend framework other than React you'll need to use [Wallet
 
 ## 2. Wrap Your App in `WalletProvider`
 
-Next, you'll wrap your `App` with `<WalletProvider>` to give all your components access to useful data, hooks, and utilities. You'll also need to pass in information about Xpla networks, such as the mainnet or chainId, into the provider via `getChainOptions`.
+Next, you'll wrap your `App` with `<WalletProvider>` to give all your components access to useful data, hooks, and utilities. You'll also need to pass in information about XPLA Chains, such as the mainnet or chainId, into the provider via `getChainOptions`.
```
Comment (jhlee-young) [inline content/docs/develop/tools/wallet-provider/get-started-with-wallet-provider.md]: XPLA Chain networks? I think XPLA Chain is a name, which cannot be plural

File: content/docs/full-node/manage-a-validator/validator-faq.md [hunk 4]
```diff
@@ -123,17 +123,17 @@ A validator operator's "self-bond" refers to the amount of XPLA delegated to its
...
 
 Ask your validator for their xplavaloper address.
 
-Be careful when delegating to validators outside of the active set. Some inactive validators may be jailed or are no longer supported. Xpla Vault only displays active or recently active validators who are able to participate in consensus.
+Be careful when delegating to validators outside of the active set. Some inactive validators may be jailed or are no longer supported. XPLA Vault only displays active or recently active validators who are able to participate in consensus.
 
 ### Is There a Faucet?
 
-Use the [Xpla faucet](https://faucet.xpla.io/) to obtain coins for the testnet.
+Use the [XPLA Chain faucet](https://faucet.xpla.io/) to obtain coins for the testnet.
```
Comment (jhlee-young) [inline content/docs/full-node/manage-a-validator/validator-faq.md]: should be XPLA Faucet

File: content/docs/learn/xpla-vault/testnet.md [hunk 5]
```diff
@@ -5,15 +5,15 @@ title: Testnet
...
 
 ## Get Testnet XPLA
 1. Open web wallet and connect your extension wallet.
-2. In the bottom left corner of Xpla Vault, switch **Mainnet** to **Testnet**.
-2. Copy your wallet address. Your wallet address will appear at the top of the Xpla web wallet near your wallet name. Your wallet address will look like this: `xpla<random-numbers-and-letters>`
-3. Open your browser and navigate to the [Xpla Faucet](https://faucet.xpla.io).
-4. Paste your Xpla address in the **Testnet address** field, select XPLA, and click **Send me tokens**.
+2. In the bottom left corner of XPLA Vault, switch **Mainnet** to **Testnet**.
+2. Copy your wallet address. Your wallet address will appear at the top of the XPLA web vault near your wallet name. Your wallet address will look like this: `xpla<random-numbers-and-letters>`
+3. Open your browser and navigate to the [XPLA Chain Faucet](https://faucet.xpla.io).
```
Comment (jhlee-young) [inline content/docs/learn/xpla-vault/testnet.md]: please update faucet name here as well

File: content/docs/develop/core-modules/wasm.md [hunk 6]
```diff
@@ -11,19 +11,19 @@ The WASM module implements the execution environment for WebAssembly smart contr
...
 
 - A unique **contract address** with an account that holds funds.
 - A **code ID**, where its logic is defined.
 - Its own **key-value store**, where it can persist and retrieve data.
 
 #### Contract Address
 
-Upon instantiation, each contract is automatically assigned a Xpla account address, called the contract address. The address is procedurally generated on-chain without an accompanying private and public key pair, and it can be completely determined by the contract's number order of existence. For instance, on two separate Xpla networks, the first contract will always be assigned the address `Xpla18vd8fpwxzck93qlwghaj6arh4p7c5n896xzem5`, and similarly for the second, third, and so on.
+Upon instantiation, each contract is automatically assigned a XPLA Chain account address, called the contract address. The address is procedurally generated on-chain without an accompanying private and public key pair, and it can be completely determined by the contract's number order of existence. For instance, on two separate XPLA Chains, the first contract will always be assigned the address `xpla18vd8fpwxzck93qlwghaj6arh4p7c5n896xzem5`, and similarly for the second, third, and so on.
```
Comment (jhlee-young) [inline content/docs/develop/core-modules/wasm.md:22]: XPLA Chain networks

File: content/docs/develop/tools/wallet-provider/get-started-with-wallet-provider.md (hunk 3 above)
Comment (caramis) [reply content/docs/develop/tools/wallet-provider/get-started-with-wallet-provider.md]: Fixed to "XPLA Chain".

File: content/docs/develop/tools/xpla-js/_index.md (hunk 1 above)
Comment (caramis) [reply content/docs/develop/tools/xpla-js/_index.md:7]: Fixed !

File: content/docs/full-node/manage-a-validator/validator-faq.md (hunk 4 above)
Comment (caramis) [reply content/docs/full-node/manage-a-validator/validator-faq.md]: fixed !

File: content/docs/learn/xpla-vault/testnet.md (hunk 5 above)
Comment (caramis) [reply content/docs/learn/xpla-vault/testnet.md]: fixed !

**Review by jhlee-young — APPROVED** [review]
LGTM 👍

**Pull request merged**